apps/src/code-studio/components/progress/LessonProgress.jsx

```diff
@@ -83,7 +83,9 @@ class LessonProgress extends Component {
     lessonTrophyEnabled: PropTypes.bool,
     width: PropTypes.number,
     setDesiredWidth: PropTypes.func,
-    currentPageNumber: PropTypes.number
+    // currentPageNumber is intentionally a string since it gets used
+    // as a string in the URL
+    currentPageNumber: PropTypes.string
```

Comment:
It looks like this gets compared to another number https://github.com/code-dot-org/code-dot-org/blob/f4a46c17366ffa4c08eb937e05bd1aacddc9893b/apps/src/code-studio/components/progress/LessonProgress.jsx#L209 (unless that's also a string) Seems like it would make more sense to leave this as a number especially since the variable name has Number in it

Reply:
This is a good question. The page number is conceptually similar to a sub-level ID. This was why I decided to swap it to a string. That being said, I don't think we ever use it in a way that would care whether it's a number or a string. So it very well may make sense to just change it back to being a number.

Reply:
yeah i think that makes sense. nice catch @maureensturgeon!